src/Workspaces/CSharp/Portable/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -159,6 +159,15 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 }
             }
 
+            // * { - in the switch expression context
+            if (currentToken.IsKind(SyntaxKind.OpenBraceToken) && currentToken.Parent.IsKind(SyntaxKind.SwitchExpression))
```

Comment:
```suggestion
            if (currentToken.IsKind(SyntaxKind.OpenBraceToken) && currentToken.Parent.IsKind(SyntaxKindEx.SwitchExpression))
```

Reply:
Ah, I see. That's the trick ;)

Thnx, I'll incorporate all the suggestions now.